tools/apiview/emitters/typespec-apiview/package.json

```diff
@@ -53,19 +53,19 @@
     "dist/**",
     "!dist/test/**"
   ],
-  "dependencies": {
-    "@azure-tools/typespec-azure-core": "0.29.0",
-    "@azure-tools/typespec-autorest": "0.29.0",
-    "@typespec/compiler": "0.43.0",
-    "@typespec/http": "0.43.1",
-    "@typespec/rest": "0.43.0",
-    "@typespec/versioning": "0.43.0"
+  "peerDependencies": {
+    "@azure-tools/typespec-azure-core": ">=0.26 <1.0",
+    "@azure-tools/typespec-autorest": ">=0.26 <1.0",
+    "@typespec/compiler": ">=0.40 <1.0",
+    "@typespec/http": ">=0.40 <1.0",
+    "@typespec/rest": ">=0.40 <1.0",
```

Comment:
I believe `@typespec/http` and `@typespec/rest` can be moved to `devDependency` since they are only used in test code